analyzer/src/funTest/assets/projects/external/git-repo-expected-output.yml

```diff
@@ -4095,18 +4095,26 @@ analyzer:
       NPM::isarray:2.0.5:
       - timestamp: "1970-01-01T00:00:00Z"
         source: "NPM"
-        message: "The package-lock.json file was created with an old version of npm,\n\
-          so supplemental metadata must be fetched from the registry.\nThis is a one-time\
-          \ fix-up, please be patient...\nminimatch@0.3.0: Please update to minimatch\
-          \ 3.0.2 or higher to avoid a RegExp DoS issue"
+        message: "The package-lock.json file was created with an old version of npm,\
```

Comment:
commit-msg-title:  maybe say "improve mapping of lines to issues" ,
because "Group related lines when parsing issues" to me just indicates that grouping is changed while I believe the "split-up" of issues is of similar importance.

Is "Tests need to be adjusted to be invariant against the order of issues as
different NPM versions / on different platforms the issue order may vary." now obsolete? (e.g. belongs to the split-out code no more part of this commit?)

Reply:
> to me just indicates that grouping is changed while I believe the "split-up" of issues is of similar importance.

To me, splitting into issues also is grouping, but I've changed the text anyway.

> e.g. belongs to the split-out code no more part of this commit?

Yes, thank, I've removed that paragraph.